src/melee/it/id.h

```diff
@@ -0,0 +1,278 @@
+#ifndef _melee_it_id_h_
+#define _melee_it_id_h_
+
+// Based on "ID Lists" on the SSBM datasheet
+// https://docs.google.com/spreadsheets/d/1JX2w-r2fuvWuNgGb6D3Cs4wHQKLFegZe2jhbBuIhCG8
+enum {
+    It_Kind_Capsule = 0x00,
+    It_Kind_Box = 0x01,
+    It_Kind_Taru = 0x02, // Barrel
+    /*
+    0x03, // Egg
+    0x04, // Party Ball (Kusudama)
+    0x05, // Barrel Cannon (TaruCann)
+    0x06, // Bob-omb (BombHei)
+    0x07, // Mr. Saturn (Dosei)
+    0x08, // Heart Container
+    0x09, // Maxim Tomato
+    0x0A, // Starman (Super Star)
+    0x0B, // Home-Run Bat
+    0x0C, // Beam Sword
+    0x0D, // Parasol
+    0x0E, // Green Shell (G Shell)
+    0x0F, // Red Shell (R Shell)
+    0x10, // Ray Gun (L Gun)
+    0x11, // Freezie (Freeze)
+    0x12, // Food
+    0x13, // Proximity Mine (MSBomb)
+    0x14, // Flipper
+    0x15, // Super Scope (S Scope)
+    0x16, // Star Rod
+    0x17, // Lip's Stick
+    0x18, // Fan (Harisen)
+    0x19, // Fire Flower (F Flower)
+    0x1A, // Super Mushroom (Kinoko)
+
+    0x1D, // Warp Star (WStar)
+    0x1E, // Screw Attack (ScBall)
+    0x1F, // Bunny Hood (RabbitC)
+    */
+    It_Kind_MetalB = 0x20, // Metal Box
+    /*
+    0x21, // Cloaking Device (SpyCloak)
+    0x22, // Poké Ball (M Ball)
+
+    // ITEM RELATED
+    0x23, // Ray Gun recoil effect
+    0x24, // Star Rod Star
+    0x25, // Lips Stick Dust
+    0x26, // Super Scope Beam
+    0x27, // Ray Gun Beam
+    0x28, // Hammer Head
+    0x29, // Flower
+    0x2A, // Yoshi's egg (Event)
+
+    // MONSTERS
+    0x2B, // Goomba (DKuriboh)
+    0x2C, // Redead (Leadead)
+    0x2D, // Octarok (Octarock)
+    0x2E, // Ottosea
+    0x2F, // Stone(Octarok Projectile)
+
+    // CHARACTER RELATED
+    */
+    It_Kind_MarioFireball = 0x30, // Mario's fireball special
+    /*
+    0x31, // MAYBE Dr. Mario's pill?
+    0x32, // Kirby's Cutter beam
+    0x33, // Kirby's Hammer
+    0x34, //
+    0x35, //
+    0x36, // Fox's Laser
+    0x37, // Falco's Laser
+    0x38, // Fox's shadow
+    */
+    It_Kind_FalcoShadow = 0x39, // Falco's shadow
```

Comment:
Call it AfterImage. That's the internal name in structs and makes more sense than Shadow which would be their floor shadow, if you didn't know better.